README.md

```diff
@@ -2,127 +2,156 @@
 
 ## Description
 
-VTEX App that shows a collection of products.
+VTEX App that shows a shelf of products, and this app is used by store theme.
```

Comment:
The VTEX shelf app is a store component that shows a collection of products, and this app is used by store theme. :hammer: